app/src/main/java/net/tensory/rxjavatalk/data/BattleFrontFeed.java

```diff
@@ -7,16 +7,14 @@
 import net.tensory.rxjavatalk.models.House;
 import net.tensory.rxjavatalk.models.HouseBattleResult;
 
-import java.util.Arrays;
-import java.util.List;
 import java.util.concurrent.ThreadLocalRandom;
 import java.util.concurrent.TimeUnit;
 
 import io.reactivex.Observable;
 import io.reactivex.subjects.BehaviorSubject;
 
 public class BattleFrontFeed {
-    private static final int EMIT_RATE_SECONDS = 10;
+    private static final int EMIT_RATE_SECONDS = 16;
```

Comment:
WIth the additional text, it's really hard to humanly consume as fast as the feed is going